CHANGELOG.md

```diff
@@ -9,8 +9,44 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added
 
+- Metric `tnt_sharded_queue_api_role_stats` is a summary with quantiles of
+  `sharded_queue.api` role API calls (#71).
+  The metric includes a counter of API calls and errors, see:
+  https://github.com/tarantool/metrics/?tab=readme-ov-file#summary
```

Comment:
Since changelog is markdown, you can make it more readable with hyperlink.

Reply:
It's also better to use documentation link, it's pretty good and more useful

https://www.tarantool.io/en/doc/latest/book/monitoring/api_reference/#summary

Reply:
Thank you, updated.